src/main/java/com/datalogics/pdf/samples/manipulation/RemoveInteractivity.java

```diff
@@ -127,6 +128,10 @@ public static void removeInteractivity(final URL inputUrl, final URL outputUrl)
         try {
             pdfDoc = DocumentUtils.openPdfDocument(inputUrl);
 
+            // Before removing any interactive elements, generate their appearances using the AppearanceService
+            // to ensure that the document has all required appearances so the document looks correct at the end.
+            AppearanceService.generateAppearances(pdfDoc, null, null);
```

Comment:
I agree that passing null for the ``apContext`` is the best thing to do here, but Patrick's question is a reasonable one—one that customers studying this sample are likely to ask as well. Perhaps we should mention this reason in the code comment?